scripts/defaults.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
This script requires Python 3<b>.8</b> (for the f"-strings and dirs_exist_ok), but this isn't stated anywhere. Please add either a comment to the README.md or refactor it, so that any python 3 (the lowest supported AFAIK is Python 3.5) version is suitable. I'd personally prefer the latter option, because I for example have Python 3.5 installed.

Reply:
I'll make it compatible with Python 3.7.3 because this is the version shipped in debian stable which should be a good indicator of what is available on most systems.

https://packages.debian.org/buster/python3

Reply:
23101e69a4df6c7ca3e22817625d2fe627818d37